src/components/UI/organisms/MenuItem/MenuItem.tsx

```diff
@@ -0,0 +1,49 @@
+import './MenuItem.css';
+
+import { Box, Button, Modal, TextField, Typography } from '@mui/material';
+import { useState } from 'react';
+
+function MenuItem() {
+  const [amount, setAmount] = useState(0);
+  const [open, setOpen] = useState(false);
+  const handleOpen = () => setOpen(true);
+  const handleClose = () => setOpen(false);
+
+  const inc = () => {
```

Comment:
This method name doesn't bring any knowledge about its usage. So ... what it is for?